ivy/functional/ivy/experimental/manipulation.py

```diff
@@ -1981,3 +1981,68 @@ def unique_consecutive(
         counts=ivy.array([2, 2, 1, 2, 1]))
     """
     return ivy.current_backend(x).unique_consecutive(x, axis=axis)
+
+
+@handle_exceptions
+@handle_nestable
+@handle_array_like_without_promotion
+@to_native_arrays_and_back
+@handle_array_function
+def trim_zeros(
+    filt: Union[
+        List[Union[ivy.Array, ivy.NativeArray]],
+        Sequence[Union[ivy.Array, ivy.NativeArray]],
+    ],
+    /,
+    *,
+    trim: Optional[str] = "fb",
+) -> Union[
+    List[Union[ivy.Array, ivy.NativeArray]],
+    Sequence[Union[ivy.Array, ivy.NativeArray]],
+]:
+    """
+    Trim the leading and/or trailing zeros from a 1-D array or sequence.
+
+    Parameters
+    ----------
+    filt
+        input array.
+
+    trim
+        A string with ‘f’ representing trim from front and ‘b’ to trim from back.
+        Default is ‘fb’, trim zeros from both front and back of the array.
+
+    Returns
+    -------
+    ret
+        The result of trimming the input. The input data type is preserved.
```

Comment:
How is this preserved? Clearly it is converted into an `ivy.Array` with an according backend data. I don't think we support native list/tuple as `.data` for an `ivy.Array`, so it's best if we remove this clause. Whoever have to do the `numpy` or `jax` frontend will have to convert it back to a native list/tuple.

Reply:
I've removed the clause